core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-card/feedback-improvement-card.directive.ts

```diff
@@ -40,7 +40,8 @@ angular.module('oppia').directive('feedbackImprovementCard', [
                 text: latestMessage.text,
                 author: latestMessage.author_username,
                 updatedOn: latestMessage.created_on,
-                updatedStatus: latestMessage.updated_status,
+                updatedStatus: (
+                  numMessages === 1 ? null : latestMessage.updated_status),
```

Comment:
Shouldn't a thread with one message have a status? Not sure why the "1" case needs to be handled differently.